src/pages/skillsmapSearch.js

```diff
@@ -40,11 +42,23 @@ const SkillsmapSearchPage = ({
   }
 
   return (
-    <main>
-      <h1>Skillsmap Search</h1>
-      <p>Skillsmap Search Text ...</p>
-
-      <div>
+    <Layout>
+      <H1WithSubheader
+        headerText="Choose your subjects"
+        subheaderText="Choose a subject to see the skills you are building in it. Choose up to
+        3 subjects to see the overlap of skills between them."
+      />
+      <div
+        style={{
+          width: '100%',
+          display: 'grid',
+          gridTemplateColumns:
+            'repeat(auto-fill, minmax(var(--skills-button-width), var(--skills-button-width)))',
+          gridGap: 'var(--spacing-m)',
+          paddingTop: 'var(--spacing-m)',
+          paddingBottom: 'var(--spacing-m)',
+        }}
+      >
```

Comment:
for components which are only used on that page, but are a bit big and ugly, you could define it at the top of the file and then call it in the return

Reply:
e.g. 

`
const SubjectGrid = ({children}) => <div style={{ ... }}>{children}</div>
`